hugo/assets/scss/components/teaser.scss

```diff
@@ -11,9 +11,7 @@
     background-color: var(--teaser-background, transparent);
     color: var(--text-color);
     height: 100%;
-    padding: $p-gutter;
     position: relative;
-    transition: background-color 0.2s;
```

Comment:
this is needed for the normal teaser which does have a background change on focus

Reply:
There doesn't seem to be a background-color change on focus in .teaser though